potpie/cli/auth/gitlab_commands.py

```
@@ -0,0 +1,292 @@
+
+def _print_issue_row(row: dict[str, Any]) -> None:
+    iid = _esc(row.get("iid"))
+    title = _esc(row.get("title"))
+    print_plain_line(f"\n  #{iid}  {title}".rstrip(), as_json=False)
+    if row.get("assignee"):
+        print_plain_line(f"    Assignee: {_esc(row.get('assignee'))}", as_json=False)
+    if row.get("labels"):
+        labels = row.get("labels") or []
+        if isinstance(labels, list) and labels:
+            print_plain_line(
+                f"    Labels: {', '.join(str(l) for l in labels)}", as_json=False,
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)